lib/Bio/EnsEMBL/DataCheck/Checks/XrefVersions.pm

```diff
@@ -0,0 +1,51 @@
+=head1 LICENSE
+
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::XrefVersions;
```

Comment:
This datacheck is not required, it is redundant. There is already a generic datacheck that tests for empty strings in all NULLable fields: https://github.com/Ensembl/ensembl-datacheck/blob/release/101/lib/Bio/EnsEMBL/DataCheck/Checks/BlankNulls.pm